buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -756,15 +754,15 @@ class BeamModulePlugin implements Plugin<Project> {
       // Enables a plugin which performs code analysis for common bugs.
       // This plugin is configured to only analyze the "main" source set.
       if (configuration.enableFindbugs) {
-        project.apply plugin: 'findbugs'
-        project.findbugs {
+        project.apply plugin: 'com.github.spotbugs'
+        project.spotbugs {
           excludeFilter = project.rootProject.file('sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml')
           sourceSets = [sourceSets.main]
         }
-        project.tasks.withType(FindBugs) {
+        project.tasks.withType(com.github.spotbugs.SpotBugsTask) {
           reports {
-            html.enabled = false
-            xml.enabled = true
+            html { enabled = true }
```

Comment:
definitely+1 on having decent developer defaults.

Unfortunately the Jenkins plugin requires xml output. And as findbugs/spotbugs support only single report format and I further was under the impression that findbugs should be removed, I simply switch to xml. Of course, this is a different story now.

Regarding that parameter, do we have other places where we have different configuration on jenkins? If so, would it make sense to extract some Jenkins-plugin which collects all this differences, so that we only either autodetect, we run on Jenkins (by checking some Jenkins env-var) or set only a single parameter, ciBuild=true or such.